### DISCUSSION TITLE
Failed to run `config` for lualine.nvim

### DISCUSSION THREAD

**Discussion by W-quan** [LazyVim/LazyVim#5220] [Q&A]
Failed to run `config` for lualine.nvim

...local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/ui.lua:108: attempt to index global 'Snacks' (a nil value)

# stacktrace:
  - /LazyVim/lua/lazyvim/plugins/ui.lua:108 _in_ **values**
  - /LazyVim/lua/lazyvim/config/init.lua:217
  - /LazyVim/lua/lazyvim/config/init.lua:213 _in_ **setup**
  - /LazyVim/lua/lazyvim/init.lua:7 _in_ **setup**
  - ~/.config/nvim/lua/config/lazy.lua:17
  - ~/.config/nvim/init.lua:2

**Comment by dpetka2001** [comment]
Did you disable Snacks?

**Reply by W-quan** [reply]
No

**Reply by folke** [reply]
Then you're not using LazyVim, or you manually added `import` in your `lazy.lua`, but in the incorrect order.

**Reply by W-quan** [reply]
```
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not (vim.uv or vim.loop).fs_stat(lazypath) then
  local lazyrepo = "https://github.com/folke/lazy.nvim.git"
  local out = vim.fn.system({ "git", "clone", "--filter=blob:none", "--branch=stable", lazyrepo, lazypath })
  if vim.v.shell_error ~= 0 then
    vim.api.nvim_echo({
      { "Failed to clone lazy.nvim:\n", "ErrorMsg" },
      { out, "WarningMsg" },
      { "\nPress any key to exit..." },
    }, true, {})
    vim.fn.getchar()
    os.exit(1)
  end
end
vim.opt.rtp:prepend(lazypath)

require("lazy").setup({
  spec = {
    -- add LazyVim and import its plugins
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    -- import/override with your plugins
    { import = "plugins" },
  },
  defaults = {
    -- By default, only LazyVim plugins will be lazy-loaded. Your custom plugins will load during startup.
    -- If you know what you're doing, you can set this to `true` to have all your custom plugins lazy-loaded by default.
    lazy = false,
    -- It's recommended to leave version=false for now, since a lot the plugin that support versioning,
    -- have outdated releases, which may break your Neovim install.
    version = false, -- always use the latest git commit
    -- version = "*", -- try installing the latest stable version for plugins that support semver
  },
  install = { colorscheme = { "tokyonight", "habamax" } },
  checker = {
    enabled = true, -- check for plugin updates periodically
    notify = false, -- notify on update
  }, -- automatically check for plugin updates
  performance = {
    rtp = {
      -- disable some rtp plugins
      disabled_plugins = {
        "gzip",
        -- "matchit",
        -- "matchparen",
        -- "netrwPlugin",
        "tarPlugin",
        "tohtml",
        "tutor",
        "zipPlugin",
      },
    },
  },
})
```

Copy from Starter. Didn't change anything.  This error happen after upgrade 14(may be 13)

**Reply by dpetka2001** [reply]
The error couldn't happen in 13 because Snacks wasn't part of LazyVim then. And I just did a fresh LazyVim installation from scratch and didn't get this error. It should be something on your end.

**Reply by W-quan** [reply]
How can i debug this error?

**Reply by dpetka2001** [reply]
This error doesn't happen with a fresh LazyVim installation, like shown in the following screencast

[Screencast_2024-12-23-10-47-55.webm](https://github.com/user-attachments/assets/e8fda60c-d457-4c2e-b781-b9c0d9b1b470)

So, I would suggest to start fresh with a new LazyVim installation without any of your personal configuration and then gradually move over small pieces of your personal configuration until you find the culprit. 

It's not possible for us to debug your configuration. You'll have to put in some effort and try to solve this from your side, unless you can give exact reproducible steps.

**Reply by W-quan** [reply]
Hey friends.  There is the problem:

```
  {
    "sainnhe/sonokai",
    opts = {
      sonokai_style = "atlaantis",
    },
  },

  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "sonokai",
    },
  },
```

I have change the theme. but why?
